<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LabelDefinitionEditor</class>
 <widget class="QDialog" name="LabelDefinitionEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>459</width>
    <height>460</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Label Definition Editor</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Label Name:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="labelName"/>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>13</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Label Properties</string>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Paper Size:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="paperSize"/>
          </item>
         </layout>
        </item>
        <item row="1" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Columns:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="labelColumns">
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>16777215</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="2" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Rows:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="labelRows">
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>16777215</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="3" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Width:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="labelWidth">
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>16777215</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="4" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>Height:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="labelHeight">
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>16777215</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="5" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <item>
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>Offset X:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="labelStartXOffset">
            <property name="maximum">
             <number>16777215</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="6" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_8">
          <item>
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>Offset Y:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="labelStartYOffset">
            <property name="maximum">
             <number>16777215</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="7" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_9">
          <item>
           <widget class="QLabel" name="label_9">
            <property name="text">
             <string>Horizontal Gap:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="labelHorizontalGap">
            <property name="maximum">
             <number>16777215</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="8" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_10">
          <item>
           <widget class="QLabel" name="label_10">
            <property name="text">
             <string>Vertical Gap:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="labelVerticalGap">
            <property name="maximum">
             <number>16777215</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="1">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="btnCancel">
         <property name="text">
          <string>Cancel</string>
         </property>
         <property name="autoDefault">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnSave">
         <property name="text">
          <string>Save</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>258</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
